docs/install.rst

```diff
@@ -21,7 +27,13 @@ The easiest way to install Python and SciPy is with `Anaconda`_, a free scientif
 
 #. Create a new environment for Pylake named pylake by invoking the following command::
 
-    conda create -n pylake
+    conda create -n pylake conda=4.9
+
+.. note::
+
+    On Windows, there are issues with OpenSSL that preclude a successful installation on the latest version of conda.
+    This is why it is recommended to create an environment specifically with conda version `4.9` for now.
+    See `OpenSSL Error`_ for more information.
```

Comment:
hmm, this seems to break the numbering in the ordered list. I think it can be solved by indenting the note block ([from here](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/lists.html#code-images-and-other-content-inside-lists)

![image](https://user-images.githubusercontent.com/61475504/222100505-0441f418-4035-4a42-a636-e76bbb75987a.png)